src/robot/libdoc.py

```diff
@@ -107,6 +107,8 @@
 
   python -m robot.libdoc src/MyLibrary.py doc/MyLibrary.html
   jython -m robot.libdoc MyLibrary.java MyLibrary.html
+  python -m robot.libdoc src/MyLibrary.py doc/MyLibrary.json
+  python -m robot.libdoc src/MyLibrary.json doc/MyLibrary.html
```

Comment:
Probably `doc/MyLibrary.json` would be better as the source in the latter example?